lldb/include/lldb/Host/HostInfoBase.h

```diff
@@ -147,6 +147,7 @@ class HostInfoBase {
       FileSpec &lldb_shlib_spec, FileSpec &file_spec, bool verify) {
     return false;
   }
+  static std::vector<std::string> GetLibrarySearchPaths() { return {}; }
```

Comment:
Can you add a doxygen comment explaining the use-case and expecations here>

Reply:
Sure!